packages/muelu/adapters/CMakeLists.txt

```diff
@@ -61,7 +60,7 @@ APPEND_SET(HEADERS
   tpetra/MueLu_TpetraOperatorAsRowMatrix.hpp
 )
 
-IF (${PACKAGE_NAME}_ENABLE_Belos AND ${PACKAGE_NAME}_ENABLE_Ifpack2 AND Tpetra_INST_INT_INT AND ${PACKAGE_NAME}_ENABLE_Experimental)
+IF (${PACKAGE_NAME}_ENABLE_Belos AND ${PACKAGE_NAME}_ENABLE_Ifpack2 AND HAVE_TEUCHOS_COMPLEX)
```

Comment:
Are we effectively moving the ShiftedLaplacian stuff out of EXPERIMENTAL? Or has that always been the case and this test now reflects that?

Reply:
Currently, on the one hand, `MueLu`'s `CMake` logic protects this particular line with a condition that we must be in an experimental build. But, on the other hand, `MueLu`'s `CMake` logic is such that the ETI and tests for `ShiftedLaplacian` get added to the build, even when we're not experimental. And so I made this change to align everything on `ShiftedLaplacian` not being in experimental. Do you agree with this change? Or would you prefer that we align everything the other way around and move everything related to `ShiftedLaplacian` to experimental?

Reply:
I'm cool with it not being experimental.